### ISSUE TITLE
Wrong logic: Augmentation only should be in Train Data

### ISSUE THREAD

**Issue by hvt1609** [skanderhamdi/attention_cnn_lstm_covid_mel_spectrogram#2]
```
images,covid_status = shuffle(images,covid_status, random_state=75)
(trainX,testX,trainY,testY) = train_test_split(images,covid_status,test_size=0.2,shuffle=True)
```

Augmentation only should be in Train Data. You use augmentation before you split the train-test => many samples in the test are the same as with Train data. Your model is easy to learn and sure that your accuracy is very high.

If my comment is right, please update your paper with a good split.

Thank.

**Comment by skanderhamdi** [comment]
Hello, thank you for the note,
You may be right, however, our point of using augmented data for training, validation and testing is related to the limited number of positive instances, and also, we are sure there's no training data on the testing set as we did not obtained an overfitting case, but we will take this in consideration and let the testing set as it is and make some new experiments, if we see that is really affect our results, we'll publish an update.

Many thanks for the note sir.
